src/components/NewMessage/NewMessageNewFileDialog.vue

```diff
@@ -212,6 +220,7 @@ export default {
 			}
 
 			await shareFile(filePath, this.token, '', '')
+			this.loading = false
```

Comment:
There is an early return when there is an error so it will stay loading forever.

Reply:
right, will add there as well